layout/module/__module__/__fileset__/ingest/pipeline.yml.tpl

```diff
@@ -53,6 +53,51 @@ processors:
         field: destination.as.organization_name
         target_field: destination.as.organization.name
         ignore_missing: true
+  - append:
```

Comment:
I think it's better to populate `related.hosts` in `liblogparser.js`, as the other related fields.